sdk/openai/Azure.AI.OpenAI/src/Custom/OpenAIClient.cs

```diff
@@ -17,13 +17,33 @@ namespace Azure.AI.OpenAI
     /// <summary> Azure OpenAI APIs for completions and search. </summary>
     public partial class OpenAIClient
     {
+        private const string publicOpenAIVersion = "1";
+        private const string publicOpenAIEndpoint = $"https://api.openai.com/v{publicOpenAIVersion}";
+
+        private string PublicOpenAIToken { get; } = "";
+
+        /// <summary> Initializes a instance of OpenAIClient using the public OpenAI endpoint. </summary>
+        /// <param name="token"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="token"/> is null. </exception>
```

Comment:
Per the previous comment, I'm (perhaps excessively) paranoid about averting the potential confusion between AOAI AzureKeyCredential and this non-Azure OpenAI API key. Can we add a remark to help? Like:

```
        <remarks>
            This key is used to connect to a non-Azure OpenAI resource created directly with OpenAI. For Azure OpenAI resources as created and maintained in Azure Portal, use a constructor that provides an Azure resource endpoint and credential, instead.
        </remarks>
```

Reply:
remark added